Automatic Validation ended with: 
> 2026-01-15T01:27:20.1574382Z ##[error] Manifest is invalid. manifests/l/lucasdeeiroz/RobotRunner/2.0.87/lucasdeeiroz.RobotRunner.locale.en-US.yaml

(Automated response - build 1197.)

--- a/manifests/l/lucasdeeiroz/RobotRunner/2.0.87/lucasdeeiroz.RobotRunner.locale.en-US.yaml
+++ b/manifests/l/lucasdeeiroz/RobotRunner/2.0.87/lucasdeeiroz.RobotRunner.locale.en-US.yaml
@@ -14,7 +14,6 @@ ShortDescription: A modern, cross-platform GUI for Robot Framework and Appium, d
 ReleaseNotesUrl: https://github.com/lucasdeeiroz/robot_runner/releases/tag/v2.0.87
 ReleaseDate: 2026-01-14
 Documentations:
-ReleaseDate: 2026-01-14
 - DocumentLabel: Wiki
   DocumentUrl: https://github.com/lucasdeeiroz/robot_runner/wiki
 ManifestType: defaultLocale

